connectivity/check/check.go

```diff
@@ -35,6 +35,9 @@ type Parameters struct {
 	Debug                 bool
 	PauseOnFail           bool
 	SkipIPCacheCheck      bool
+	Perf                  bool
+	Duration              int
```

Comment:
Change this to `time.Duration` (ref. my comment on the CLI flags definition):

```suggestion
	Duration              time.Duration
```